### DISCUSSION TITLE
Can't discover modelfiles anymore

### DISCUSSION THREAD

**Discussion by zono50** [open-webui/open-webui#796] [General]
# Bug Report - Can't discover modelfiles anymore

## Description - Anytime i try to get a modelfile, it takes me to a blank create a modelfile page. This only happened after running latest docker pull

**Bug Summary:**
[Provide a brief but clear summary of the bug]

Anytime i try to get a modelfile, it takes me to a blank create a modelfile page.

**Steps to Reproduce:**
1) Discover new modelfiles
2) click get button
3) type http://localhost:3000 and import
4) shows blank modelfile page.

**Expected Behavior:**
I expect to download and use the new modelfile

**Actual Behavior:**

## Environment

Manjaro linux latest version

When i click import after putting in localhost:3000 url, i get the following 

![image](https://github.com/open-webui/open-webui/assets/8764705/edb547ba-212e-40de-b938-56401c2b24fc)

**Comment by tjbck** [comment]
Hi, could you verify that you're using open-webui container image and not the outdated ollama-webui one? Thanks!

Here's our migration guide: https://github.com/open-webui/open-webui#moving-from-ollama-webui-to-open-webui

**Comment by zono50** [comment]
I followed the migration guide, but I think i screwed up cause my login wouldn't work anymore, so created a new login, and all my stuff is gone, and still can't discover new model files

**Comment by zachferr** [comment]
+1 on the original issue, but I only experience it in Orion(MacOS, webkit based browser), Chrome works fine.
